client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -133,6 +134,12 @@ const PluginsBrowser = ( {
 
 	const selectedSite = useSelector( getSelectedSite );
 	const sitePlan = useSelector( ( state ) => getSitePlan( state, selectedSite?.ID ) );
+	const installedPlugins =
+		useSelector( ( state ) => getInstalledPlugins( state, [ selectedSite?.ID ] ) ) || [];
+
+	const activePlugins =
+		installedPlugins.filter( ( activePlugin ) => activePlugin.sites[ selectedSite?.ID ].active ) ||
+		[];
```

Comment:
Nitpicking: The [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) method will always return an array, so I don't see the need of `|| []`. 

```suggestion
	const activePlugins =
		installedPlugins.filter( ( activePlugin ) => activePlugin.sites[ selectedSite?.ID ].active );
```

---

PS: This case is different from the `installedPlugins` above as `useSelector` is being used, therefore you can't be sure an array will be returned as you can with `filter`.